src/helpers/formatters.ts

```diff
@@ -1,5 +1,9 @@
 import { DateTime } from 'luxon';
 
+export const stringToMarkup = (string: string): { __html: string } => {
+  return { __html: string };
```

Comment:
We could use [isomorphic-dompurify](https://github.com/kkomelin/isomorphic-dompurify) here to give us that extra reassurance.